public/userInfo.php

```diff
@@ -294,7 +294,7 @@ function resize() {
                 $userRank = $userMassData['Rank'];
                 $rankPct = sprintf("%1.2f", ($userRank / $countRankedUsers) * 100.0);
                 $rankOffset = (int) (($userRank - 1) / 25) * 25;
-                echo "<a href='/globalRanking.php?s=5&t=2&o=$rankOffset'>$userRank</a> / $countRankedUsers ranked users (Top $rankPct%)";
```

Comment:
`s` could also be eliminated here, but it's probably best to leave the two links on the profile with explicit parameters.

Reply:
I've restored the explicit parameters on this page in https://github.com/RetroAchievements/RAWeb/pull/1598/commits/6735af1497c47b674df7727a2357a9de39143328. For my own intellectual curiosity, could you help me better understand the motivation behind keeping them on the user profile page?

Reply:
These are explicit links to show the user's position in the list. We want to ensure they always go to the user even if the defaults change (not that I'm expecting them to change).